api/apiclient.go

```diff
@@ -1247,7 +1247,7 @@ func isX509Error(err error) bool {
 // object id, and the specific RPC method. It marshalls the Arguments, and will
 // unmarshall the result into the response object that is supplied.
 func (c *conn) APICall(facade string, vers int, id, method string, args, response interface{}) error {
-	err := c.client.Call(rpc.Request{
+	err := c.client.Call(context.TODO(), rpc.Request{
```

Comment:
This will come from APICall, but to do that is about 1 million breaking changes, so let's do that in another PR.